Feature/color dummy

little-pinecone opened the pull request.

little-pinecone committed:
Change RandomService::nextFloat() behaviour. Fix documentation.

little-pinecone committed:
Add ColorDummy.

little-pinecone committed:
Fix SonarQube code smells, add EqualsVerifier.

daniel-frak commented on src/main/java/dev/codesoapbox/dummy4j/RandomService.java:
```diff
@@ -163,10 +162,10 @@ public double nextDouble(double lowerBound, double upperBound) {
      */
     public float nextFloat() {
-        return random.nextFloat();
+        return this.nextFloat(1);
```
`this` is unnecessary here.

daniel-frak commented on src/main/java/dev/codesoapbox/dummy4j/dummies/color/NumberFormatter.java:
```diff
@@ -0,0 +1,22 @@
+ */
+final class NumberFormatter {
+
+    static final DecimalFormat DECIMAL = new DecimalFormat("0.##");
```
Should be private.

daniel-frak commented on pom.xml:
```diff
@@ -80,6 +80,13 @@
         </dependency>
+
+        <!--https://mvnrepository.com/artifact/nl.jqno.equalsverifier/equalsverifier-->
+        <dependency>
```
Should be `<scope>test</scope>`

little-pinecone committed:
Implement code review suggestions and update README.md.

Fix NumberFormatter::DECIMAL visibility, set equalsverifier scope to test, remove unnecessary code, add Color to dummies in README.md and index.md.
